NAMESPACE

```diff
@@ -65,6 +66,8 @@ export(removeRowsWithNAInSpecCol)
 export(returnColsWithMoreThanFiftyCategories)
 export(selectData)
 export(skip_on_not_appveyor)
+export(startProdLogs)
+export(stopProdLogs)
 export(variationAcrossGroups)
```

Comment:
Should we standardize to the new function underscore format (to keep PR consistent)?

Reply:
Yeah, I did it. I forgot that functions are underscores as well.